backend/manager/modules/restapi/types/src/main/java/org/ovirt/engine/api/restapi/types/StorageLogicalUnitMapper.java

```diff
@@ -105,13 +112,17 @@ public static StorageServerConnections map(LogicalUnit logicalUnit, StorageServe
         }
         if (logicalUnit.isSetPort()) {
             entity.setPort(logicalUnit.getPort().toString());
+        } else {
+            // Setting default port to please the StorageLogicalUnitMapperTest#testRoundtrip test
+            entity.setPort("3260");
```

Comment:
would be better to either use an existing constant like [this](https://github.com/oVirt/ovirt-engine/blob/d63b345767f58365e0280cd55ed82b22263ec257/backend/manager/modules/restapi/jaxrs/src/main/java/org/ovirt/engine/api/restapi/resource/BackendHostResource.java#L92) or to introduce another constant